pkg/reconciler/source/rabbitmqsource.go

```diff
@@ -154,18 +154,6 @@ func (r *Reconciler) reconcileRabbitObjects(ctx context.Context, src *v1alpha1.R
 	src.Status.MarkSecretReady()
 
 	if src.Spec.RabbitmqResourcesConfig.Predeclared {
-		_, err := r.rabbit.RabbitmqV1beta1().Exchanges(src.Namespace).Get(ctx, src.Spec.RabbitmqResourcesConfig.ExchangeName, metav1.GetOptions{})
```

Comment:
I think it might be important to fix this separately and quickly. Anything predeclared won't work with this block here unless they're using topology-operator.